gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFSInputStream.java

```diff
@@ -126,7 +127,9 @@ public synchronized int read(@Nonnull byte[] buf, int offset, int length) throws
 
   @Override
   public synchronized void seek(long pos) throws IOException {
+    long startTime = System.nanoTime();
```

Comment:
why not use stopwatch?

Reply:
I did not think of using stopwatch but nanoTime seems to be fast and precise.

Reply:
I think Stopwatch might be the right choice. This is what we use at other places in the connector too.